src/components/recent-podcast/script.js

```diff
@@ -0,0 +1,4 @@
+export default {
+  name: 'recent-podcast',
+  props: ['recentTrack'],
```

Comment:
for purpose of the scope, i suggest renaming the **recentTrack**  prop to **track**

Reply:
done